crates/context/src/lib.rs

```diff
@@ -309,9 +309,10 @@ impl ContextManager {
         }
 
         handle.put(
-            &ContextIdentityKey::new(context.id, identity_secret.public_key()),
+            &ContextIdentityKey::new(context.id, identity_secret.public_key()), //local node saving its local identity
```

Comment:
artifact of development iteration? or you think this is useful because the code is unreadable?

```suggestion
            &ContextIdentityKey::new(context.id, identity_secret.public_key()),
```

Reply:
Something weird is happening, I keep commiting things I don't actually have locally...